lib/_debugger.js

```diff
@@ -1557,7 +1556,16 @@ Interface.prototype.repl = function() {
 
     // Exit debug repl
     self.exitRepl();
-  });
+    
+    self.repl.rli.removeListener('SIGINT', exitDebugRepl);
+    self.repl.removeListener('exit', exitDebugRepl);
+  }
+  
+  // Exit debug repl on Ctrl + C
+  this.repl.rli.on('SIGINT', exitDebugRepl);
+  
+  // Exit debug repl on '.exit'
+  this.repl.on('exit', exitDebugRepl);
```

Comment:
Maybe prefer `'close'`? Does the same thing but I think `'exit'` may be an older event we don't use internally.